cmd/server-main.go

```diff
@@ -482,8 +483,12 @@ func serverMain(c *cli.Context) {
 		fatalIf(errors.New("empty directory"), "Configuration directory cannot be empty.")
 	}
 
+	// Disallow relative paths, figure out absolute paths.
+	configDirAbs, err := filepath.Abs(configDir)
```

Comment:
you can assign it back to `configDir` or make this into a block so that `configDirAbs` is available further below.